config/environment_indicator.switcher.local.yml

```
@@ -0,0 +1,11 @@
+uuid: 0de89751-6687-422f-b307-eba74064d7c4
+langcode: en
+status: true
+dependencies: {  }
+machine: local
+description: null
+name: Local
+weight: ''
+url: 'https://numbers.test'
+fg_color: '#000000'
+bg_color: '#ffffff'
```

```suggestion
fg_color: '#ffffff'
bg_color: '#000000'
```

Has this black-on-white been deployed many places for locals? If possible I'd prefer to keep it as core has: with white-on-black.